server/lib/enrich/article.js

```diff
@@ -31,6 +31,13 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	content.hasGraphics = content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0 ? true : false;
```

Comment:
If you're wondering why the `? true : false`, it's because `content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0` returns 0 when `graphics: 0`. Would `!!` be a solution here?

Reply:
no need to use a ternary there:

```> content = {contentStats :{graphics: 0}}
{ contentStats: { graphics: 0 } }
> content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0
0
> content.contentStats && content.contentStats.graphics > 0
false
```
It returns `0` because `somethingTruthy && 0` is `0`


This would work
```
content.contentStats && content.contentStats.graphics > 0
```

Reply:
So if there's no `graphics` key, that would be `false`, right?

Reply:
You can test it easily in your node console :) 

```
> content.contentStats && content.contentStats.graphics > 0
false
> content.contentStats && content.contentStats.SOMETHINGELSE > 0
false
> undefined > 0
false
```

Reply:
It still feels like code is sometimes pure magic 😂